src/components/ModalViewContainer/index.tsx

```diff
@@ -27,86 +58,40 @@ const useStyles = makeStyles((theme) => ({
     marginLeft: theme.spacing(2),
     marginRight: theme.spacing(2),
     marginTop: theme.spacing(0),
-    marginBottom: theme.spacing(2),
+    marginBottom: theme.spacing(2)
     // paddingLeft: theme.spacing(0),
     // paddingRight: theme.spacing(0),
     // paddingTop: theme.spacing(0),
     // paddingBottom: theme.spacing(0),
   }
-
 }));
 
-
-const ModalViewContainer = (props) => {
+const ModalViewContainer = props => {
   const classes = useStyles();
 
   const routingInfoRef = useRef({});
-  const {
-    getPConnect,
-    routingInfo,
-    loadingInfo
-  } = props;
+  const { getPConnect, routingInfo, loadingInfo } = props;
+  const pConn = getPConnect();
   const {
     CONTAINER_TYPE: { MULTIPLE },
     // eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
     PUB_SUB_EVENTS: { EVENT_SHOW_CANCEL_ALERT }
   } = PCore.getConstants();
-  const pConn = getPConnect();
+  const { subscribe, unsubscribe } = PCore.getPubSubUtils();
   const [bShowModal, setShowModal] = useState(false);
-  const [bSubscribed, setSubscribed] = useState(false);
   const [bShowCancelAlert, setShowCancelAlert] = useState(false);
   const [oCaseInfo, setOCaseInfo] = useState({});
   const [createdView, setCreatedView] = useState<any>(null);
   // eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
-  const [templateName, setTemplateName] = useState("");
-  const [title, setTitle] = useState("");
+  const [templateName, setTemplateName] = useState('');
+  const [title, setTitle] = useState('');
   const [arNewChildrenAsReact, setArNewChildrenAsReact] = useState<Array<any>>([]);
-  const [itemKey, setItemKey] = useState("");
+  const [itemKey, setItemKey] = useState('');
   // eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
```

Comment:
It looks like this eslint directive isn't necessary any more. So it should be removed.